### ISSUE TITLE
Phase/Quest/Faction related portals are visible for everyone

### ISSUE THREAD

**Issue by 55Honey** [azerothcore/azerothcore-wotlk#5771]
#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance
- [x] Horde

##### CONTENT PHASE:

- [x] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

##### CURRENT BEHAVIOUR:

Portals which should not be visible to the other faction or should be only temporarily visible during certain quests/phases, are visible for everyone.

This involves the following object GUIDs, possibly more:
- 2133392
- 2133393
- 2133394
- 2133395

##### EXPECTED BLIZZLIKE BEHAVIOUR:

The portal to UC in the Stormwind Keep near the king, should not be visible for every Alliance player at all times.

##### STEPS TO REPRODUCE THE PROBLEM:

1. `.go o 2133395`
2. Find the portal to UC visible, no matter your GM state or quest state

##### EXTRA NOTES:

Something might have broken the invisibilityType check.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/6947789622de45479d2311fd5d1635af3a957b5d

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Kitzunu** [comment]
https://github.com/azerothcore/azerothcore-wotlk/issues/5773

**Comment by Si1ker** [comment]
Build: core/db from eca3dc2d

2133392 - Bug confirmed - Portal phase 1
2133393 - Working - Phase 64
2133394 - Working - Phase 64
2133395 - Bug confirmed - Portal phase 1

- Looking at the base data, they come with those phases, (so be in phase 1 seems intended.)
- The values stored in current sql base data regarding those portals comes from:
https://github.com/azerothcore/azerothcore-wotlk/blob/4b7454574412cfe23bb5902b9b3d42d2313b3541/data/sql/archive/3.x/db_world/2019_03_26_01.sql#L40

On gameobject_addon, they all have the same values regarding visibility but Idk what those values mean as tc does not have the table as far as I can see and AC does not have a description for those.
```sql
SELECT * FROM gameobject_addon WHERE guid IN (
2133392,
2133393,
2133394,
2133395);
```

![image](https://user-images.githubusercontent.com/61223313/117727706-a3289c80-b1a5-11eb-9696-485bbcd9a76a.png)

**Comment by Kitzunu** [comment]
cc @UltraNix

**Comment by pangolp** [comment]
Conformed, I just updated the emulator.
And the portal continues to appear.

![WoWScrnShot_051521_184200](https://user-images.githubusercontent.com/2810187/118379038-153a2080-b5ae-11eb-94fe-1bd92b1d76ac.jpg)

Shouldn't this portal be invoked by Lady Jaina Proudmoore?

**Comment by UltraNix** [comment]
Fixed by #5885.
